strictdoc/export/html/form_objects/grammar_form_object.py

```diff
@@ -28,22 +28,33 @@ def __init__(
         field_mid: str,
         field_name: str,
         document_mid: str,
+        field_human_title: Optional[str] = None,
     ):
         self.field_mid: str = field_mid
         self.field_name: str = field_name
         self.document_mid: str = document_mid
+        self.field_human_title: Optional[str] = field_human_title
 
     @staticmethod
-    def create(*, grammar_element: GrammarElement, document_mid: str):
+    def create(
+        *,
+        grammar_element: GrammarElement,
+        document_mid: str,
+        field_human_title: Optional[str] = None,
```

Comment:
This `field_human_title` argument does not seem to be used anywhere. Let's remove it.